docs/network-flow-visibility.md

```diff
@@ -202,30 +203,56 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
-please update the `databaseURL` following
-[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1).
-
-The ClickHouse credentials are used in the Flow Aggregator. Them are also specified
-in `flow-aggregator.yml` as a Secret named `clickhouse-secret` as shown below.
-Please make the corresponding changes if you change the ClickHouse credentials.
-
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  labels:
-    app: flow-aggregator
-  name: clickhouse-secret
-  namespace: flow-aggregator
-stringData:
-  password: clickhouse_operator_password
-  username: clickhouse_operator
-type: Opaque
-```
-
-Please refer to the [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
-to learn about more ClickHouse configuration options in Flow Aggregator.
+The ClickHouse TCP service is used by the Flow Aggregator. If you have changed
+the TCP port, please update the `clickHouse.databaseURL` in the Flow Aggregator
+Helm Chart values.
+
+The ClickHouse credentials are used in the Flow Aggregator. If you have changed
+the ClickHouse credentials, please update the `clickHouse.connectionSecret` in the
+Flow Aggregator Helm Chart values.
+
+Please refer to the Flow Aggregator Helm Chart [README](../build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+to learn more about ClickHouse configuration options in Flow Aggregator.
+
+##### ClickHouse Cluster
+
+From Theia v0.2.0, we support deploying a ClickHouse cluster with data replication.
+A ClickHouse cluster consists of one or more shards. Shards refer to the servers
+that contain different parts of the data. You can deploy multiple shards to scale
+the cluster horizontally. Each shard consists of one or more replica hosts. You
+can deploy multiple replicas in a shard to ensure reliability.
+
+To deploy ClickHouse as a cluster, please set `clickhouse.cluster.enable` to true
+and set `clickhouse.cluster.shards` and `clickhouse.cluster.replicas` per your
+requirement.
+
+ClickHouse uses [Apache ZooKeeper](https://zookeeper.apache.org/) for storing
+replicas meta information. By default, a ZooKeeper cluster using `emptyDir` with
+3 replicas will be deployed for ClickHouse cluster. The 3 replicas are expected
+to be deployed on 3 different Nodes. To use a customized ZooKeeper cluster,
+please refer to the
+[ZooKeeper setup instructions for ClickHouse](https://github.com/Altinity/clickhouse-operator/blob/master/docs/zookeeper_setup.md)
+and set `clickhouse.cluster.zookeeperHosts` to your ZooKeeper hosts.
+
+The default affinity allows only one ClickHouse instance per Node. Please make
+sure there are at least 4 Nodes available in your Kubernetes cluster to deploy
+a 2-shard-2-replica cluster with this affinity. To change the affinity, please
```

Comment:
In the doc you did not mention before that the default values are 2 shard/2 replicas. It might be worth doing that as user might wonder why we're talking about this.

Also, due to this antiaffinity policy, can we have a single "local" PV for all instances?

Reply:
Updated, thanks! The PV problem is discussed and updated as https://github.com/antrea-io/theia/pull/55#discussion_r922450358